test/Parse/operator_decl_designated_types.swift

```diff
@@ -33,11 +33,15 @@ infix operator **+> : MediumPrecedence, UndeclaredProtocol
 // expected-error@-1 {{use of undeclared type 'UndeclaredProtocol'}}
 
 prefix operator *+*> : MediumPrecedence
+// expected-error@-1 {{use of undeclared type 'MediumPrecedence'}}
```

Comment:
I think this is a regression, `MediumPrecedence` is declared right above on line 7. Right?

Reply:
Yes, I think so.